programs/uxd/tests/unit_tests/state/alloyx_vault_depository.rs

```diff
@@ -7,7 +7,7 @@ mod test_alloyx_vault_depository {
 
     #[test]
     fn test_alloyx_vault_depository_space() -> Result<()> {
-        assert_eq!(ALLOYX_VAULT_DEPOSITORY_SPACE, 1149);
+        assert_eq!(ALLOYX_VAULT_DEPOSITORY_SPACE, 1152);
```

Comment:
1024+128=1152, should be reasonably aligned?

Reply:
haha yes! I don't even know the impact tbh